EngineLayer/ProteinParsimony/ProteinParsimonyEngine.cs

```diff
@@ -23,7 +25,7 @@ public class ProteinParsimonyEngine : MetaMorpheusEngine
         /// </summary>
         private readonly bool _treatModPeptidesAsDifferentPeptides;
 
-        public ProteinParsimonyEngine(List<PeptideSpectralMatch> allPsms, bool modPeptidesAreDifferent, CommonParameters commonParameters, List<string> nestedIds) : base(commonParameters, nestedIds)
+        public ProteinParsimonyEngine(List<PeptideSpectralMatch> allPsms, bool modPeptidesAreDifferent, CommonParameters commonParameters, List<(string fileName, CommonParameters fileSpecificParameters)> fileSpecificParameters, List<string> nestedIds) : base(commonParameters, fileSpecificParameters, nestedIds)
```

Comment:
again not sure here if both commonParameters and the list of file specific common parameters are required

Reply:
same answer